src/api/HttpLink.js

```diff
@@ -19,10 +27,34 @@ export default ({
 		}
 	};
 
-	// Use the regular HttpLink if batching is disabled.
-	if (!apolloBatching) {
-		return new HttpLink(options);
-	}
+	const stellateOptions = {
+		...options,
+		uri: stellateGraphqlUri ?? uri,
+	};
+
+	const link = split(
+		operation => {
+			// check to see if we have a Stellate uri available
+			if (!stellateGraphqlUri) {
+				return false;
+			}
+			console.log('- - - - - - - - - -');
```

Comment:
Could we use logFormatter for these logs?

Reply:
I don't intend to leave these in. It's just for running locally and verifying the behavior. I think these in particular would just be noisy but if there is something you think we should log intentionally I can add those.